src/main/java/com/northconcepts/datapipeline/foundations/examples/jdbc/GenerateJavaBeansFromADatabase.java

```diff
@@ -0,0 +1,145 @@
+package com.northconcepts.datapipeline.foundations.examples.jdbc;
+
+import com.northconcepts.datapipeline.foundations.jdbc.JdbcConnection;
+import com.northconcepts.datapipeline.foundations.jdbc.JdbcTable;
+import com.northconcepts.datapipeline.foundations.jdbc.JdbcTableColumn;
+import com.northconcepts.datapipeline.internal.lang.Util;
+
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+public class GenerateJavaBeansFromADatabase {
+
+    public static void main(String... args) {
+
+        JdbcConnection connection = new JdbcConnection()
+                .setDriverClassName("org.postgresql.Driver")
+                .setUrl("jdbc:postgresql://localhost:5432/customers")
+                .setUsername("username")
+                .setPlainTextPassword("password");
+
+        for(JdbcTable table : connection.loadTables().getTables()) {
+
+            Catalog catalog = new Catalog();
```

Comment:
It looks like we're creating a catalog for each table (above) and adding one table to each catalog's table (below).  Is this the intent?  

Let's discuss more tomorrow.  The JavaBeans we need to create are like JPA -- one per table -- not replicating what's already in the JdbcXXX classes.  Sorry for the misunderstanding.

Reply:
updated the PR